### PR TITLE
Fix bug in update_channel method

### PR DESCRIPTION
**Pull request by Elleppi**

### CODE REVIEW

File: src/iotics/lib/grpc/iotics_api.py [hunk 1]
```diff
@@ -130,18 +130,41 @@ def update_channel(self, channel: typing.Optional[grpc.Channel] = None):
     def _update_api(self, api: ApiBase, channel: grpc.Channel):
         if isinstance(api, TwinApi):
             self._twin_api = TwinApi(self._auth, channel)
+            self.create_twin = self.twin_api.create_twin
+            self.describe_twin = self.twin_api.describe_twin
+            self.delete_twin = self.twin_api.delete_twin
+            self.list_twins = self.twin_api.list_twins
+            self.update_twin = self.twin_api.update_twin
+            self.upsert_twin = self.twin_api.upsert_twin
```
Comment (Evolter) [inline src/iotics/lib/grpc/iotics_api.py:138]: If we would extract those to separate methods, we could call it here and in the init - something like:
```python
self._update_twin_apis()
self._update_feed_apis()
```

**Review by Evolter — APPROVED** [review]
Small suggestion, but LGTM.

It would be nice to have a test case for the bug as well.

**Review by bernardcooke-iotics — APPROVED** [review]
Another suggestion but not mandatory to fix the bug - good catch :tada:

File: src/iotics/lib/grpc/iotics_api.py (hunk 1 above)
Comment (Elleppi) [reply src/iotics/lib/grpc/iotics_api.py]: Thank you both.
I adopted Adrian's solution as it solves the bug and the initialisation of the APIs.
I've also applied a little change to the same Class, let me know what you think.
@Evolter regarding the test case: considering there's at the moment no test suite in the library and I need this bug to be fixed ASAP, I was wondering whether we could have a separate a ticket merely focused on the creation of a test suite with several scenarios

Comment (Evolter) [reply src/iotics/lib/grpc/iotics_api.py]: So I thought - I'm happy for the test to be added at later date. LGTM

Comment (Elleppi) [reply src/iotics/lib/grpc/iotics_api.py]: Great. I'm going to merge this PR and create a ticket for the Tests

Comment (Elleppi) [reply src/iotics/lib/grpc/iotics_api.py]: https://ioticlabs.atlassian.net/browse/EI-1553

**Pull request merged**